app/src/main/java/com/example/music_app/domain/RequestTokenUseCase.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.music_app.domain
+
+import com.example.music_app.data.LoginRepository
+
+class RequestTokenUseCase {
+    private val loginRepository: LoginRepository = LoginRepository()
+
+    suspend fun requestToken(code: String) = loginRepository.requestToken(code)
```

Comment:
I believe this could be changed to operator `invoke` override like
```
    operator fun invoke(date: Date): String {
        return formatter.format(date)
    }
```